src/backend/OverlayFS.ts

```diff
@@ -96,9 +151,15 @@ export default class OverlayFS extends file_system.SynchronousFileSystem impleme
     return true;
   }
 
-  public _syncSync(file: preload_file.PreloadFile<any>): void {
+  public _syncAsync(file: PreloadFile<UnlockedOverlayFS>, cb: (err: ApiError)=>void): void {
+    this.createParentDirectoriesAsync(file.getPath(), () => {
```

Comment:
Once you add it, check error argument from `createParentDirectoriesAsync` before proceeding.